inst/WORDLIST

```diff
@@ -20,8 +13,4 @@ customizable
 filesystem
 hostname
 json
-lubridate
-runApp
 sqlfile
```

Comment:
remove **sqlfile** from Wordlist. 
Reason:No need to ignore because it is replaced by SQLite file in R/data-storage-sqlite.R